QATCH/VisQAI/src/view/main_window.py

```
@@ -115,65 +119,208 @@ def __init__(self, parent=None):
+            elif status == LicenseStatus.ACTIVE:
+                self.setCentralWidget(self.tab_widget)
+                expiration_str = license_data.get('expiration', '')
+                if expiration_str:
+                    try:
+                        expiration_date = dt.datetime.fromisoformat(
+                            expiration_str)
+                        days_remaining = (expiration_date -
+                                          dt.datetime.now()).days
+                        self._update_status_bar(
+                            f"Licensed: {days_remaining} days remaining", permanent=True)
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)